tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -198,5 +244,26 @@ Feature: Duplicate product from Back Office (BO).
     And product copy_of_product1 should have identical customization fields to product1
     And product copy_of_product1 should have 1 customizable text field
     And product copy_of_product1 should have 0 customizable file fields
-#@todo: assert stock info
+    And product "copy_of_product1" should have following stock information:
+      | minimal_quantity    | 12         |
+      | quantity            | 0          |
+      | location            |            |
```

Comment:
Location and quantity are not the same ?

`available_now_labels` & `available_later_labels` are not duplicated ?

Reply:
yes, by default these fields seems to not be duplicated in old page, as well as whole stock_available relation.
We need to clarify which fields should be duplicated and which not. (ping @MatShir). 
Btw Im not sure if we should wait for these new specifications or we should merge this PR as its only testing the current default behavior. (so when the new specs comes out, we will adjust the code and adjust the tests)